lib/node_modules/@stdlib/array/log/examples/index.js

```diff
@@ -0,0 +1,32 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+var discreteUniform = require( '@stdlib/random/base/discrete-uniform' ).factory;
+var naryFunction = require( '@stdlib/utils/nary-function' );
+var filledBy = require( '@stdlib/array/filled-by' );
+var map = require( '@stdlib/utils/map' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var log = require( '@stdlib/array/log' );
```

Comment:
Use the relative path here. Apart from doctests, this should be the only difference between the example as shown in the README and the example as included in this file.